ceagle/api/v1/status.py

```diff
@@ -29,9 +32,17 @@ def get_status(period):
 
 
 @bp_status.route("/health/<period>")
-@fake_status.get_status_health
```

Comment:
I would recommend to keep this decorator but apply changes related to this decorator as first commit, so both real and fake code could be tested normally in unit tests in this commit